Makefile

```diff
@@ -84,6 +87,9 @@ test-isort: ## run isort checks
 test-mypy: ## run type tests
 	mypy ${SRC_FILES_PROD}
 
+test-format: ## Run code formatting tests
```

Comment:
Please add this target to `make test-quality`, such that it is run in CI.

Reply:
Sure.